templates/artist/artist_detail.html

```diff
@@ -200,6 +226,69 @@ <h1 class="header-font">{{ artist.name }}</h1>
 {% block extrajs %}
     <script src="https://checkout.stripe.com/checkout.js"></script>
     <script type="text/javascript">
+
+    <script type="text/javascript" src="{% static "js/vendor/smooth-scroll.js" %}"></script>
+    <script type="text/javascript">
+
+
+
+        var doughnutData = [
+                {
+                    value: {{ campaign.fans_percentage }},
+                    color: "#46BFBD",
+                    highlight: "#5AD3D1",
+                    label: "Fan %"
+                },
+                {
+                    value: {{ campaign.artist_percentage }},
+                    color:"#F7464A",
+                    highlight: "#FF5A5E",
+                    label: "Artist %"
+                }
+
+            ];
+
+        var randomScalingFactor = function(){ return Math.round(Math.random()*100)};
+        var lineChartData = {
+            labels : ["1x","2x","3x","4x","5x"],
```

Comment:
Can you walk me through this graph? I'm confused by the labels and the data.
In the first line you have `percentage_roi:100, 200, 400, ...` but the labels says `1x, 2x, 3x, ...`.